src/option_parser.c

```diff
@@ -268,10 +269,9 @@ struct section *get_section(const char *name)
 
 void add_entry(const char *section_name, const char *key, const char *value)
 {
-        struct section *s = get_section(section_name);
-        if (!s)
-                s = new_section(section_name);
+        struct section *s = get_or_create_section(section_name);
 
+        LOG_D("\t%s=%s", key, value);
```

Comment:
This, combined with the new logging message in `load_ini_file()` makes for a cheap way to see how dunst views the combined ini-files internally.